### PR TITLE
Misc updates 0806

### PR DESCRIPTION
**Pull request by koreno**

### CODE REVIEW

File: easypy/colors.py [hunk 1]
```diff
@@ -334,7 +336,7 @@ def colorize(text):
 
     def _subfunc(match_obj):
         colorizer = Colorizer.from_markup(match_obj.group(1))
-        return colorizer(match_obj.group(2) or match_obj.group(3))
+        return colorizer(next(filter(None, match_obj.groups()[1:])))
```
Comment (YuvalEvron) [inline easypy/colors.py]: Is it possible to get a `StopIteration` exception here (less than 2 groups)?

Comment (koreno) [reply easypy/colors.py]: no, the regex has 4 groups (some may be none).
and the regex wouldn't match if at least one of them isn't populated.

**Comment by koreno** [comment]
@idanarye ?

**Comment by idanarye** [comment]
Still think the change to `DataSize.__str__` is wrong.

**Comment by koreno** [comment]
> Still think the change to `DataSize.__str__` is wrong.

I've compensated with a new `.render` and `.__format__` implementation

**Pull request merged**